src/App.jsx

```diff
@@ -72,6 +73,7 @@ function App() {
             <Route path="/about" element={<About />} />
             <Route path="/contact-us" element={<ContactUs />} />
             <Route path="/login" element={<LogIn />} />
+            <Route path="/documentation" element={<Documentation />} />
```

Comment:
Can you add a comment to remind us to comment this route out before deploying?

Reply:
Done!